src/components/QuranReader/index.tsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 import useTranslation from 'next-translate/useTranslation';
 import { shallowEqual, useSelector } from 'react-redux';
 
-import ContextMenu from './ContextMenu';
+import ContextMenu from './ContextMenu/index';
```

Comment:
nit-picking: no need for `index` in the path as it will be resolved automatically.